tests/integration/specs_tests.rs

```diff
@@ -0,0 +1,292 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+use std::collections::HashSet;
+
+use deno_terminal::colors;
+use serde::Deserialize;
+use test_util::tests_path;
+use test_util::PathRef;
+use test_util::TestContextBuilder;
+
+#[test]
+fn test_specs() {
```

Comment:
Obviously just running this in a test is not ideal because rust captures the output so we can't see how this test is progressing without using `-- --nocapture`. This is just a super rough first pass in order to enable these kind of tests.

Reply:
We should definitely disable the rust harness for this and just use `main` -- it should be a straightforward change and we'd just need to add one more test target.

Reply:
Oh, I didn't realize how easy doing this was. Thanks!

Reply:
Wow, filtering is so easy to implement as well. Pretty sweet.